tests/installation/select_patterns_and_packages.pm

```diff
@@ -204,12 +205,17 @@ sub run {
         }
     }
     $self->package_action;
-    $secondrun++;
-    $self->gotopatterns;
-    $self->package_action;
-    $secondrun--;
-    $self->gotopatterns;
-    $self->package_action('unblock');
+    if (is_sle('15+') and check_var('PATTERNS', 'all')) {
+        record_soft_failure "bsc#1084064 - Cloud patterns conflicts";    # skip second & third runs
```

Comment:
Would it somehow be possible to actually detect the situation? Here we will not see when the bug got fixed and the soft failure will be preserved, right?